tools/config_updater.py

```diff
@@ -646,3 +678,6 @@ def main(unused_argv):
 
 if __name__ == "__main__":
   flags.StartMain(main)
+  # Wait for threads to finish inserting into the datastore
+  time.sleep(.5)
```

Comment:
I had originally planned on doing that, but what I noticed was that Flush() had already  been called so calling Flush() wouldn't solve the problem.  The flusher thread is already doing the database inserts when the interpreter starts shutting down and pulls the rug out of under it. The exceptions I was occasionally seeing with config_updater previously were in the datastore in Execute() when something went to None before it finished inserting.